src/include/parser/postgresparser.h

```diff
@@ -78,106 +95,105 @@ class PostgresParser {
    * @param root list of parsed nodes
    * @return SQLStatementList corresponding to the parsed node list
    */
-  static std::vector<std::unique_ptr<parser::SQLStatement>> ListTransform(List *root);
+  static void ListTransform(ParseResult *parse_result, List *root);
 
   /**
    * Transforms a single node in the parse list into a terrier SQLStatement object.
    * @param node parsed node
    * @return SQLStatement corresponding to the parsed node
    */
-  static std::unique_ptr<parser::SQLStatement> NodeTransform(Node *node);
+  static std::unique_ptr<SQLStatement> NodeTransform(ParseResult *parse_result, Node *node);
```

Comment:
Update documentation to include `parse_result`

Reply:
Done